app/src/pages/inside/projectSettingsPageContainer/content/elements/ruleList/ruleList.jsx

```diff
@@ -22,33 +22,45 @@ import styles from './ruleList.scss';
 
 const cx = classNames.bind(styles);
 
-export const RuleList = ({ data, actions, onToggle, disabled, ruleItemContent }) => {
+export const RuleList = ({
+  data,
+  actionsConfig,
+  onToggle,
+  disabled,
+  ruleItemContent,
+  customNode,
+  ...rest
+}) => {
   const Content = ruleItemContent;
+  const CustomNode = customNode;
   return (
     <div className={cx('container')}>
       {data.map((item) => (
         <RuleItem
           key={item.id}
           item={item}
-          actions={actions}
+          actions={actionsConfig}
           onToggle={onToggle}
           disabled={disabled}
-          content={ruleItemContent && <Content item={item} />}
+          content={ruleItemContent ? <Content item={item} {...rest} /> : null}
+          customNode={customNode ? <CustomNode item={item} {...rest} /> : null}
```

Comment:
Can we avoid spreading a lot of props here?
I think it can be done on plugin side by wrapping components.